nntrainer/dataset/databuffer_file.h

```diff
@@ -46,6 +47,14 @@ class DataBufferFromDataFile final : public DataBuffer {
    */
   DataBufferFromDataFile() : DataBuffer(DatasetType::FILE) {}
 
+  /**
+   * @brief     Constructor
+   */
+  DataBufferFromDataFile(const std::string &path) : DataBufferFromDataFile() {
```

Comment:
the argument `path` is ignored?

Reply:
This is intended. this will be removed so, no need to set path

Reply:
so the argument will be removed, or this struct?

Reply:
This struct will be removed